app/components/transfer/navigation/TransferNavigation.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { defineMessages, intlShape } from 'react-intl';
+import styles from './TransferNavigation.scss';
+import TransferNavButton from './TransferNavButton';
+
+const messages = defineMessages({
+  fromDaedalus: {
+    id: 'transfer.navigation.fromDaedalus',
+    defaultMessage: '!!!Transfer funds from Daedalus',
+  },
+  fromYoroi: {
+    id: 'transfer.navigation.fromYoroi',
+    defaultMessage: '!!!Transfer funds from another wallet',
+  },
+});
+
+type Props = {|
+  isActiveNavItem: Function,
```

Comment:
Try and avoid `Function` as the type (multiple places in this PR). Easy to make a typo in a case like this in particular